miden-lib/asm/faucets/basic.masm

```diff
@@ -0,0 +1,91 @@
+# BASIC FUNGIBLE FAUCET CONTRACT
+# =================================================================================================
+# This is a basic fungible faucet smart contract. 
+#
+# It allows the owner of the faucet to mint, distribute, and burn tokens. Token metadata is stored 
+# at position 1 as [token_symbol, decimals, 0, max_supply], where:
+# - token_symbol is expressed as <TODO add explainer>.
+# - decimals are the decimals of the token.
+# - max_supply is the maximum supply of the token.
+use.miden::sat::account
+use.miden::sat::asset
+use.miden::sat::faucet
+use.miden::sat::tx
+use.miden::eoa::basic->basic_eoa
+
+# CONSTANTS
+# =================================================================================================
+#
+# Slot in memory to store the metadata
+const.METADATA_SLOT=1
```

Comment:
nit: I'd remove `#` from line 18 to create a break in comments between section separator and the comment for `METADATA_SLOT`.

Also, the comment should be "Slot in account storage at which the metadata is stored".